tests/test_detect_target.py

```diff
@@ -93,39 +98,41 @@ def test_single_bus_image(self,
         """
         Bus image.
         """
-        # Setup
-        expected = cv2.imread(IMAGE_BUS_ANNOTATED_PATH)
-        assert expected is not None
-
         # Run
         result, actual = detector.run(image_bus)
+        detections = actual.detections
 
         # Test
         assert result
         assert actual is not None
+        assert detections is not None
     
-        error = rmse(actual, expected)
-        assert error < self.__IMAGE_DIFFERENCE_TOLERANCE
+        error = 0
+
+        for i in range(0, len(detections)):
+            error += rmse([detections[i].x1, detections[i].y1, detections[i].x2, detections[i].y2], expected_bus[i])
+        assert (error / len(detections)) < self.__IMAGE_DIFFERENCE_TOLERANCE
```

Comment:
Rename `__IMAGE_DIFFERENCE_TOLERANCE` to something like `__BOUNDING_BOX_TOLERANCE` . Use a very small value (e.g. 10^{-7} ).